org.gnome.EasyTAG.yaml

```diff
@@ -0,0 +1,122 @@
+id: org.gnome.EasyTAG
```

Comment:
Please reformat the file to follow https://github.com/flathub/flathub/wiki/YAML-Style-Guide

Reply:
Fixed by commit 892dc28.